backend/clubs/views.py

```diff
@@ -4814,7 +4810,7 @@ def current(self, *args, **kwargs):
                                 - $ref: "#/components/schemas/ClubApplication"
         ---
         """
-        qs = self.get_queryset()
+        qs = self.get_queryset().prefetch_related("extensions")
```

Comment:
Should this be `prefetch_related` or `select_related`?

Reply:
I believe `prefetch_related` since `Extension` is a reverse foreign key on `ClubApplication`